### PR TITLE
Set chain id properly in blockchain node param config scripts

### PR DESCRIPTION
**Pull request by platfowner**

### CODE REVIEW

File: config_client_api_ip_whitelist.sh [hunk 1]
```diff
@@ -87,14 +90,18 @@ else
...
+
 function config_node() {
     local node_index="$1"
     local node_ip_addr=${IP_ADDR_LIST[${node_index}]}
 
     printf "\n\n<<< Configuring ip whitelist of node $node_index ($node_ip_addr) >>>\n\n"
 
     KEYSTORE_FILE_PATH="$KEYSTORE_DIR/keystore_node_$node_index.json"
-    CONFIG_NODE_CMD="node tools/api-access/$COMMAND_NODE_JS_FILE $node_ip_addr 0 keystore $KEYSTORE_FILE_PATH"
+    CONFIG_NODE_CMD="node tools/api-access/$COMMAND_NODE_JS_FILE $node_ip_addr $CHAIN_ID keystore $KEYSTORE_FILE_PATH"
```
Comment (woomurf) [inline config_client_api_ip_whitelist.sh:104]: 🙌

**Review by woomurf — APPROVED** [review]
LGTM, Thanks!

**Review by ehgmsdk20 — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the review. Now merging..

**Pull request merged**